test/contracts/apps/disputable/disputable_app.js

```diff
@@ -44,17 +46,19 @@ contract('DisputableApp', ([_, owner, agreement, anotherAgreement, someone]) =>
 
   describe('supportsInterface', () => {
     it('supports ERC165', async () => {
+      const erc165 = await ERC165.new()
       assert.isTrue(await disputable.supportsInterface(ERC165_INTERFACE), 'does not support ERC165')
 
-      assert.equal(await disputable.ERC165_INTERFACE(), ERC165_INTERFACE, 'ERC165 interface ID does not match')
-      assert.equal(await disputable.erc165interfaceID(), ERC165_INTERFACE, 'ERC165 interface ID does not match')
+      assert.equal(await erc165.interfaceID(), ERC165_INTERFACE, 'ERC165 interface ID does not match')
+      assert.equal(await erc165.ERC165_INTERFACE(), ERC165_INTERFACE, 'ERC165 interface ID does not match')
     })
 
     it('supports Aragon App interface', async () => {
```

Comment:
We should also add a suite of tests for the base `AragonApp` contract to test that it follows ERC165 and the `IAragonApp` interface.